ext/bcmath/tests/bcround_early_return.phpt

```diff
@@ -0,0 +1,62 @@
+--TEST--
+bcround() function with early return
+--EXTENSIONS--
+bcmath
+--FILE--
+<?php
+$early_return_cases = [
+    ['123', -3],
+    ['123', -4],
+    ['123.123456', -3],
+    ['123.123456', -4],
+    ['123', 1],
+    ['123.5', 1],
+    ['123.5', 2],
+    ['123.0000000000000000000001', 22],
+    ['123.0000000000000000000001', 23],
+    ['-123', -3],
+    ['-123', -4],
+    ['-123.123456', -3],
+    ['-123.123456', -4],
+    ['-123', 1],
+    ['-123.5', 1],
+    ['-123.5', 2],
+    ['-123.0000000000000000000001', 22],
+    ['-123.0000000000000000000001', 23],
+    ['0', 0],
+    ['0.0', 0],
+    ['0.0000', 0],
+    ['-0', 0],
+    ['-0.0', 0],
+    ['-0.0000', 0],
+];
+
+foreach ($early_return_cases as [$num, $precision]) {
+    echo str_pad("[{$num}, {$precision}]", 33, ' ', STR_PAD_LEFT), " => ", bcround($num, $precision, PHP_ROUND_HALF_UP), "\n";
+}
+?>
+--EXPECT--
+[123, -3] => 0
```

Comment:
I notice that the first case in all these test files is always non-padded.

Reply:
Thanks, I fixed these!

Reply:
First case still doesn't seem to be padded here?

Reply:
Oops, I forgot to fix this.

Reply:
I fixed it